### PR TITLE
Fix for library dependency problem with items during Travis build

### PR DESCRIPTION
**Pull request by wdednam**
This change is [Reviewable](https://reviewable.io/reviews/highperformancecoder/minsky/95)

### CODE REVIEW

File: model/canvas.cc [hunk 1]
```diff
@@ -395,8 +395,20 @@ namespace minsky
...
+                       {
+                          float d=sqr((*i)->x()-x)+sqr((*i)->y()-y);
+                          if (d<minD && (*i)->visible() && (*i)->contains(x,y))
+                            {
+                              minD=d;
+                              item=*i;
+                            }
+                          return false;  
+                       });
     if (!item)
```
Comment (highperformancecoder) [inline model/canvas.cc]: This statement is now redundant

File: model/canvas.cc [hunk 2]
```diff
@@ -395,8 +395,20 @@ namespace minsky
   
   ItemPtr Canvas::itemAt(float x, float y)
   {
-    auto item=model->findAny(&Group::items,
-                        [&](const ItemPtr& i){return i->visible() && i->contains(x,y);});
+    // Fix for library dependency problem with items during Travis build     
+    ItemPtr item;                    
+    auto minD=numeric_limits<float>::max();
+    model->recursiveDo(&GroupItems::items,
```
Comment (highperformancecoder) [inline model/canvas.cc]: Great - almost exactly how I would have done it! An it looks like it has fixed the problem - yay!

File: model/canvas.cc (hunk 1 above)
Comment (wdednam) [reply model/canvas.cc]: Thanks, I didn't understand why I still needed it, but on a previous iteration the code seemed to find the item everywhere on the canvas, so I added it back in, but I must have had some other bit of code that caused that. I will remove it and recommit for a pull request.

File: model/canvas.cc (hunk 2 above)
Comment (wdednam) [reply model/canvas.cc]: Thanks. I just followed your instructions, which were crystal clear, and repurposed your existing code in canvas.cc for finding the closest "in" port. I wish all bugs were this easy to fix.:)

**Pull request merged**